src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -1,24 +1,28 @@
 package namor.template.calculator;
 
-import namor.template.stack.ArrayStack;
+import namor.template.stack.Stack;
 
 public class RPNcalculator {
 
+    private Stack numbers;
```

Comment:
Where is the generic type. Specify it when declaring the generic objects. So `Stack<Double>`